src/main/webapp/i18n/en/exercise.json

```diff
@@ -102,7 +102,7 @@
             "score": "Score",
             "duration": "Duration",
             "showBuildFailed": "Show Build Failed",
-            "showFailed": "Show Failed Submissions (without Result)",
+            "showFailed": "Show Latest Submission without Result",
```

Comment:
why uppercase?
```suggestion
            "showFailed": "Show latest Submission without Result",
```

Reply:
Other labels on this site also use capital letters. Followed the convention.